lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystemE2E.cs

```diff
@@ -194,6 +194,18 @@ public void TestCreateDirectoryE2E()
             Assert.True(_adlsClient.CheckExists(dirName));
         }
 
+        [Fact(Skip = SkipMessage)]
+        public void TestIsDirectoryE2E()
+        {
+            string dirName = $"/{_defaultFolderName}";
+            string fakeDirName = $"/fakeDir";
+            _adlsClient.CreateDirectory(dirName);
+            string fileName = UploadFromString(ContentsText);
```

Comment:
supernit: try to add newlines to break out the logic. Giant block of code are difficult to follow.

Reply:
fixed

---
In reply to: [194556435](https://github.com/apache/reef/pull/1473#discussion_r194556435) [](ancestors = 194556435)